rosdep/python.yaml

```diff
@@ -1449,6 +1449,10 @@ python-scipy:
     utopic_python3: [python3-scipy]
     vivid: [python-scipy]
     vivid_python3: [python3-scipy]
+python-selenium:
+  ubuntu:
+    pip:
+      packages: [selenium]
```

Comment:
I'm pretty sure the ubuntu version is indeed ancient and pretty much non-usable at this point. Would changing the keyname to "python-selenium-latest" or "python-selenium-pip" be a better option? (then if someone wanted to use the apt-get version later, they could?)